.github/workflows/build-and-test-typescript.yml

```diff
@@ -4,6 +4,10 @@ on:
     paths:
       - typescript/**
   workflow_dispatch:
+  workflow_call:
+    secrets:
+      PVAULT_SERVICE_LICENSE:
+        required: false
```

Comment:
Shouldn't this be true?

Reply:
It should be.
I copied it from Python & Java workflow, so we should fix it there too.